cmd/kind/kind.go

```diff
@@ -76,14 +90,13 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *Flags) error {
-	level := defaultLevel
-	parsed, err := log.ParseLevel(flags.LogLevel)
-	if err != nil {
-		log.Warnf("Invalid log level '%s', defaulting to '%s'", flags.LogLevel, level)
+	// TODO: handle --loglevel which previously supported:
+	// panic, fatal, error, warning, info, debug, trace
+	if flags.Quiet {
+		globals.SetLogger(log.NoopLogger{})
```

Comment:
If I run `kind get kubeconfig-path -q`, will it suppress all output *except for* the KUBECONFIG path?

Reply:
that's the plan

Reply:
FWIW:
```
$ kind get kubeconfig-path -q
/Users/bentheelder/.kube/kind-config-kind
```